app/src/main/res/values/strings.xml

```diff
@@ -194,102 +194,102 @@
     <string name="server_confirmed">Server confirmed.</string>
-    <string name="not_available_for_pseudo_account">Not available for pseudo account</string>
-    <string name="not_available_for_misskey_account">Not available for Misskey account</string>
-    <string name="not_available_for_mastodon_account">Not available for Mastodon account</string>
-    <string name="exit_app_when_close_protected_column">Exit app when try to close protected column with back button without confirm</string>
-    <string name="resize_image">Resize attachment image (JPEG,PNG)</string>
+    <string name="not_available_for_pseudo_account">Unavailable for pseudo accounts</string>
+    <string name="not_available_for_misskey_account">Unavailable for Misskey accounts</string>
+    <string name="not_available_for_mastodon_account">Not available for Mastodon accounts</string>
+    <string name="exit_app_when_close_protected_column">Exit app when trying to close protected column with \"Back\" button without confirmation</string>
+    <string name="resize_image">Resize attachment (JPEG, PNG)</string>
     <string name="dont_resize">Don\'t resize</string>
     <string name="long_side_pixel">Resize to %1$d pixels</string>
     <string name="image_capture">Take a picture</string>
-    <string name="missing_permission_to_access_media">Missing app permission to access media.</string>
-    <string name="attachment_uploading">Media attachment  is uploading…</string>
-    <string name="attachment_uploaded">Media attachment upload completed.</string>
+    <string name="missing_permission_to_access_media">Missing media access permission.</string>
+    <string name="attachment_uploading">Uploading media attachment…</string>
+    <string name="attachment_uploaded">Uploaded.</string>
     <string name="actions_for_status">Actions for this toot</string>
     <string name="actions_for_user">Actions for this user</string>
     <string name="open_profile">Open profile</string>
     <string name="send_message">Send message</string>
     <string name="send_message_from_another_account">Send message from another account</string>
     <string name="open_profile_from_another_account">Open profile from another account</string>
-    <string name="confirm_block_user">User \"%1$s\" will be blocked. Are you sure?</string>
-    <string name="confirm_mute_user">User \"%1$s\" will be muted. Are you sure?</string>
-    <string name="confirm_delete_status">This toot will be deleted. Are you sure?</string>
-    <string name="nickname_label">Nickname and color (if specified, it\'s shown instead of full acct)</string>
-    <string name="color">Color</string>
+    <string name="confirm_block_user">Block %1$s?</string>
+    <string name="confirm_mute_user">Mute %1$s?</string>
+    <string name="confirm_delete_status">Delete toot?</string>
+    <string name="nickname_label">Nickname and colour (if specified, it\'s shown instead of full acct)</string>
+    <string name="color">Colour</string>
```

Comment:
changing from "color" to "colour" is not accepted. 

- "favourite" word is used in Mastodon official.
- "colour" word is NOT used in Mastodon official.

In this case I should obey to Mastodon wording.

Reply:
Both "favourite" and "colour" are British English. It makes sense to stick those or
"favorite" and "color".